examples/video.html

```diff
@@ -0,0 +1,62 @@
+<!--
+  Copyright 2016 Google Inc. All Rights Reserved.
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+  
+      http://www.apache.org/licenses/LICENSE-2.0
+  
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewpoer" content="width=device-width,initial-scale=1,maximum-scale=1,shrink-to-fit=no,user-scalable=no,minimal-ui"/>
+  <title>Basic Video Demo no Webgl: Omnitone</title>
+  <script src="../build/omnitone.min.js"></script>
+</head>
+<body>
+
+  <script>
+    window.addEventListener('load', function () {
+      // Set up the context and the audio element.
+      var audioContext = new AudioContext();
+      var videoElement = document.createElement('video');
+      videoElement.loop = true;
+      videoElement.autoplay = true;
+      videoElement.crossOrigin = 'anonymous';
+      videoElement.controls = true;
+      videoElement.width = 600;
+      videoElement.height = 400;
+      videoElement.src = "https://storage.googleapis.com/omnitone-demo.google.com.a.appspot.com/fuerza-imprevista-1080p-h264-aac.mp4";
```

Comment:
If you want to include a video, you have to use something publicly available (no copyright issue) and small enough to be included in this repository.